File: Morning-snacks/PerfectNumber.js
let number = prompt("Enter a number: ");

let sumOfFactors = 0;

for(let count = 1; count < number; count++){

	if (number % count == 0){

	sumOfFactors += count;

	}
}

if (number == sumOfFactors){

console.log("It is a Perfect Number!")

}else{

console.log("Oops try again, e no work")

}

File: Morning-snacks/perfect_number.py
number = int(input("Enter a number: "))

sum_of_factors = 0

for count in range(1, number, 1):

	if number % count == 0:

		sum_of_factors += count

if number == sum_of_factors:
	
	print("It is a Perfect Number!")

else:

	print("Oops try again, e no work")

		
